src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/+page.svelte

```diff
@@ -57,15 +57,7 @@
                     domain: apexDomain
                 });
             } catch (error) {
-                // apex might already be added on organization level, skip.
-                const alreadyAdded = error?.type === 'domain_already_exists';
-                if (!alreadyAdded) {
-                    addNotification({
-                        type: 'error',
-                        message: error.message
-                    });
-                    return;
-                }
+                // Apex domain creation error needs to be silent.
```

Comment:
why silent? wouldn't user find it confusing if it fails silently, there's no feedback here. cc @Meldiron wdyt?